discover/portal/articles/210-setting-up/01-system-settings/00-intro.markdown

```diff
@@ -0,0 +1,21 @@
+# System Wide Settings [](id=system-wide-settings)
+
+It can be hard to keep track of all the configuration interfaces.  Fortunately,
+the Control Panel is most often the place to look, and there's a tidy
+*Configuration* section there that houses a lot of the higher level (for
```

Comment:
Unfortunately the "Configuration" section is really a kitchen sink which contains many things that are not really configuration. In fact, almost everything related to configuration is now within System Settings. 
Taking this into account, we might want to reword this paragraph.
I hope that for 7.2 we can put some order to this.